sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -927,7 +929,8 @@ case class Cast(child: Expression, dataType: DataType, timeZoneId: Option[String
       val tz = JavaCode.global(ctx.addReferenceObj("timeZone", timeZone), timeZone.getClass)
       (c, evPrim, evNull) =>
         code"""$evPrim =
-          org.apache.spark.sql.catalyst.util.DateTimeUtils.millisToDays($c / 1000L, $tz);"""
+          org.apache.spark.sql.catalyst.util.DateTimeUtils.millisToDays(
```

Comment:
I have a little bit concern to use a method call in the generated code. This change includes # of Java bytecode to be generated while the original code uses `/` for long type.  
Do we take the readability and maintanance or smaller generated bytecode?

Reply:
The call `toMillis()` should be inlined, shouldn't it?

Reply:
I agree with you @kiszk; this probably only adds a few ops though to invoke a static method vs a division. The issue here is the size of the generated bytecode, so the implementation of `toMillis` doesn't matter nor whether it's JITted at runtime.

Reply:
Should I revert all changes in `code` blocks?

Reply:
I reverted it back to division.